### PR TITLE
Include reform

### PR DESCRIPTION
**Pull request by adrienpacifico**

### CODE REVIEW

File: openfisca_france_data/surveys.py [hunk 1]
```diff
@@ -33,22 +33,31 @@
...
+
 log = logging.getLogger(__name__)
 
 
 class SurveyScenario(AbstractSurveyScenario):
-    def init_from_data_frame(self, input_data_frame = None, input_data_frames_by_entity_key_plural = None, tax_benefit_system = None, used_as_input_variables = None,
-            year = None):
-
+    def init_from_data_frame(self, input_data_frame = None,input_data_frames_by_entity_key_plural = None,
+                             reform = None,tax_benefit_system = None, used_as_input_variables = None, year = None):
```
Comment (benjello) [inline openfisca_france_data/surveys.py]: Missing space after comma

**Comment by benjello** [comment]
Tests don't pass (look at them) but it may be a github issue since i did find the problem in the files.
Check the missing space, run the tests locally  and if they pass commit again.

**Comment by adrienpacifico** [comment]
3 errors, 1 failure that still didn't passed before my commit. 1) might be
ok ; 2) I don't know but not my fault ; 3) Easily solved ; 4) Can't say.



I don't that my PR is responsible for those errors that probably existed
before it. I commented the 4 problems.


Ran 24 tests in 93.796s


FAILED (errors=3, failures=1)


1)

AssertionError: openfisca_data_2006 cannot be found in
[u'openfisca_data_2009', u'erfs_2006', u'erfs_2007', u'erfs_2008',
u'erfs_2009']

Adrien : "I didn't run 2006 data, sorry but it should work for the one that
use it, @benjello could you check ?"

2)

  from openfisca_france_data.collection_builders.erfs_collection_builder
import build_empty_erfs_survey_collection

ImportError: cannot import name build_empty_erfs_survey_collection

Adrien : "This import is deprecated since commit
1fda33cf045d3e737afe370a36e7dd8d76d94275 that erased the function"

3)

======================================================================

ERROR: Failure: ImportError (No module named surveys)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/local/lib/python2.7/site-packages/nose/loader.py", line 420,
in loadTestsFromName

    addr.filename, addr.module)

  File "/usr/local/lib/python2.7/site-packages/nose/importer.py", line 47,
in importFromPath

    return self.importFromDir(dir_path, fqname)

  File "/usr/local/lib/python2.7/site-packages/nose/importer.py", line 94,
in importFromDir

    mod = load_module(part_fqname, fh, filename, desc)

  File
"/Users/adrienpacifico/openfisca/openfisca-france-data/openfisca_france_data/tests/test_eipp.py",
line 31, in <module>

    from openfisca_france.surveys import SurveyScenario

ImportError: No module named surveys

Adrien : "SurveyScenario doesn't exist in openfisca_france anymore except
in obsolete code in taxipp_utils, it has been moved to
openfisca_france_data. One should correct the import"

4)

======================================================================

ERROR: openfisca_france_data.tests.test_inflation.test_inflation

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/local/lib/python2.7/site-packages/nose/case.py", line 197, in
runTest

    self.test(*self.arg)

  File
"/Users/adrienpacifico/openfisca/openfisca-france-data/openfisca_france_data/tests/test_inflation.py",
line 46, in test_inflation

    survey_scenario.inflate(target_by_variable = target_by_variable)

TypeError: inflate() got an unexpected keyword argument 'target_by_variable'

Adrien : "Variable coming from space"



2015-09-16 18:42 GMT+02:00 Mahdi Ben Jelloul <notifications@github.com>:

> Tests don't pass (look at them) but it may be a github issue since i did
> find the problem in the files.
> Check the missing space, run the tests locally and if they pass commit
> again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openfisca/openfisca-france-data/pull/104#issuecomment-140798052>
> .
>



-- 
Adrien Pacifico
PhD Student at Aix-Marseille school of economics (AMSE)
IDEP, GREQAM, EHESS

**Comment by benjello** [comment]
Try make test-ci on your computer (you need to install make)

**Comment by adrienpacifico** [comment]
Do you have a link ? Didn't found it after a fast googling.

Didn't get the purpose of test-ci.


2015-09-17 0:36 GMT+02:00 Mahdi Ben Jelloul <notifications@github.com>:

> Try make test-ci on your computer (you need to install make)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openfisca/openfisca-france-data/pull/104#issuecomment-140913281>
> .
>



-- 
Adrien Pacifico
PhD Student at Aix-Marseille school of economics (AMSE)
IDEP, GREQAM, EHESS

**Comment by benjello** [comment]
@adrienpacifico:
 - Only some tests are run on travis (data). These tests appear on the [Makefile] (https://github.com/openfisca/openfisca-france-data/blob/dev/Makefile) on the target test-ci (ci=continuous integration). The results appear on travis, see [Details] (https://travis-ci.org/openfisca/openfisca-france-data/builds/80662410) button above.
 - You can run these tests on your machine by typing (see [here] (http://stackoverflow.com/questions/2556444/install-make-command-without-already-having-make-mac-os-10-5) to install make on mac OS) 
```
make test-ci
``` 
 - These tests show a strange error see my message above (there might be an error with github). Do the change I proposed to be able to push a new commit to github to trigger a new serie of test on travis to see if the tests pass.
 - I can't accept PR if they do not pass the travis tests

**Comment by benjello** [comment]
@adrienpacifico : I made the change and submitted a PR on your PR. If you accept it, it shoul migrate t here and trigger the tests.

**Comment by benjello** [comment]
@adrienpacifico I fixed it. Check your branches !

**Pull request merged**